Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/toasts/toast-list/toast-list.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Toast } from '../toast.model';
+import { ToastService } from '../toast.service';
+import { ToastComponent } from '../toast/toast.component';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-toast-list',
+  standalone: true,
+  imports: [ToastComponent],
+  templateUrl: './toast-list.component.html',
+})
+export class ToastListComponent implements OnInit, OnDestroy {
+  toasts: Toast[] = [];
+  private subscription: Subscription = new Subscription();
+
+  constructor(private toastService: ToastService) {}
+
+  ngOnInit(): void {
+    this.subscription = this.toastService.toasts$.subscribe((toasts) => {
+      this.toasts = toasts;
+    });
+  }
+
+  ngOnDestroy(): void {
+    this.subscription.unsubscribe();
```

Comment:
It's ok to unsubscribe like that, but just want ot make sure you are aware that there is another option with created a Subject called unsubscribe$, calling this.unsubscribe$.next(); this.unsubscribe$().complete(); in the ngOnDestroy() and having takeUntil(this.unsubscribe$) opterator inside pipe(). In that way you can unsubscribe from all the subjects at once and they all will receive 'complete' event which is not happenning when you use unsibscribe(). Also, you won't need to save references for all the subscriptions inside the component class (it can get confusing when you have a bunch of subscriptions)